### PR TITLE
Changes made

### PR DESCRIPTION
**Pull request by ghost**
Hi!, I added the class AmazonSearch

### CODE REVIEW

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		this.driver = driver;
+	}
+	public void clickFilterCheckBox(int noElement)
+	{
+		//pul rikoest
+		List<WebElement> myElements = driver.findElements(By.xpath("//li[starts-with(@id,'p_')]//i[@class='a-icon a-icon-checkbox']"));
+		//li[starts-with(@id,'p_')]//i[@class='a-icon a-icon-checkbox']CHECKBOS
+		//li[starts-with(@id,'n/')]//a[@class='a-link-normal s-navigation-item'] DEPARTAMENTOS		
+		//li[starts-with(@id,'p_')]//a[@class='a-link-normal s-navigation-item']
+		//*[@class='a-icon a-icon-checkbox' or @class='a-link-normal s-navigation-item']
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java]: Delete comments

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	}
+	public void clickFilterCheckBox(int noElement)
+	{
+		//pul rikoest
+		List<WebElement> myElements = driver.findElements(By.xpath("//li[starts-with(@id,'p_')]//i[@class='a-icon a-icon-checkbox']"));
+		//li[starts-with(@id,'p_')]//i[@class='a-icon a-icon-checkbox']CHECKBOS
+		//li[starts-with(@id,'n/')]//a[@class='a-link-normal s-navigation-item'] DEPARTAMENTOS		
+		//li[starts-with(@id,'p_')]//a[@class='a-link-normal s-navigation-item']
+		//*[@class='a-icon a-icon-checkbox' or @class='a-link-normal s-navigation-item']	
+		System.out.println(myElements.get(noElement).getText());
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java]: Use Log class

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        myElements.get(noElement).click();
+	}
+	public void clickElementResult(int indexElement) {
+		driver.findElement(By.xpath("//span[@class='celwidget slot=SEARCH_RESULTS template=SEARCH_RESULTS widgetId=search-results index="+indexElement+"']//img[@class='s-image']")).click();
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearch.java]: Take out the XPath declarations and move it to the upper section of the class

**Review by navi911407 — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
I have changed the things you ask me for, please let me know if it's correct uwu

**Pull request merged**